pyproject.toml

```diff
@@ -1,3 +1,29 @@
 [tool.black]
 line-length = 119
-target-version = ['py35']
\ No newline at end of file
+target-version = ['py39']
+
+[tool.poetry]
+name = "metadata"
+version = "0.1.0"
+description = "Codebase for including metadata (e.g., URLs, timestamps, HTML tags) during language model pretraining."
+authors = ["Your Name <you@example.com>"]
```

Comment:
Just a nitpick, I have the impression that it is not the same thing as what is put in `setup.py`. Is it voluntary? :slightly_smiling_face:

Reply:
Indeed, I was talking about the `author` / `authors` section.

Thanks for the link, I now understand that the section keys differ between `poetry` and `Setuptools`